operator/controllers/lifecycle/keptntask/job_utils.go

```diff
@@ -101,11 +102,11 @@ func (r *KeptnTaskReconciler) updateJob(ctx context.Context, req ctrl.Request, t
 	}
 	return nil
 }
-func (r *KeptnTaskReconciler) getJob(ctx context.Context, jobName string, namespace string) (*batchv1.Job, error) {
+func (r *KeptnTaskReconciler) getJob(ctx context.Context, task *v1alpha3.KeptnTask, namespace string) (*batchv1.Job, error) {
```

Comment:
i think here it makes more sense to keep using the `jobName` parameter, as opposed to expecting a KeptnTask to be passed